packages/key-management/src/Bip32Account.ts

```diff
@@ -33,6 +38,11 @@ export class Bip32Account {
     return key.toRawKey();
   }
 
+  async deriveMultiSigPublicKey({ role, index }: MultiSigAccountKeyDerivationPath) {
+    const key = await this.extendedAccountPublicKey.derive([KeyPurpose.MULTI_SIG, role, index]);
```

Comment:
```suggestion
```
This is wrong: scheme in cip1854 is
![image](https://github.com/input-output-hk/cardano-js-sdk/assets/2457868/5921621f-f08c-4300-a28e-3ec015ba8b5e)

While `this.extendedAccountPublicKey` is already at `purpose/coin_type/account_ix` - we can't use this key to derive new `purpose`. It should be done on another level, by creating a new `Bip32Account` with a different `extendedAccountPublicKey` that is derived with the 1854 `purpose`.

Reply:
I looked around and could not find where the purpose of 1852 was defined for `this.extendedAccountPublicKey`. I also could not find the derivation path for the first three steps in the [keyagent](https://github.com/input-output-hk/cardano-js-sdk/blob/ed9caf09bf5f980a10733ce62b1a698ead884ef4/packages/key-management/test/Bip32Account.test.ts#L17-L25) when specifying a new `Bip32Account` or the derive function. Where can a new purpose (1854) be specified for `extendedAccountPublicKey `?

Reply:
Fixed on fac71bd